lib/rubocop/options.rb

```diff
@@ -34,7 +34,7 @@ def parse(command_line_args)
     private
 
     def args_from_file
-      if File.exist?('.rubocop')
+      if File.file?('.rubocop')
```

Comment:
This fails for symbolic links and named pipes. (I'm not sure if we care.)

An alternative is:

```ruby
def args_from_file
  IO.readlines('.rubocop').map(&:strip)
rescue
  []
end
```